Workbooks/App Hub/Cosmos Databases/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Cosmos Databases",
+    "author": "Microsoft",
+    "galleries": [
+        { "type": "workbook", "resourceType": "Monitoring Groups", "order": 300 },
```

Comment:
should this be 200 to match the other order?

Reply:
They are in different galleries - it was meant to be different.